http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -787,8 +813,14 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""galaxy.rbac.enabled=false""",
       raw"""galaxy.rbac.serviceAccount=${ksa.value}""",
       raw"""rbac.serviceAccount=${ksa.value}""",
-      // TODO Update during https://broadworkbench.atlassian.net/browse/IA-2171
-      raw"""persistence={}"""
+      raw"""persistence.nfs.name=${config.galaxyAppConfig.nfsPersistenceName}""",
```

Comment:
Does it work setting `persistence.nfs.name` and `persistence.postgres.name` to a static value when there are multiple apps in a cluster? For some things we need cluster-unique names; for other things names only need to be unique within a namespace. Not sure which category this is :)